xrm/xrm.d.ts

```diff
@@ -344,6 +349,9 @@ declare namespace Xrm
         prependOrgName( sPath: string ): string;
     }
 
+    export module Context {
```

Comment:
it looked a bit excessive for me. up to you to decide to keep them or remove them.